news/2024/2024-02-05-skin-of-the-year-2023-results.md

```diff
@@ -0,0 +1,84 @@
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/FMN.jpg)
+
+The winner of "Icons of an Era", [Spoo](https://osu.ppy.sh/users/11805037), claims their spot on this list with their stunning entry, [FOOL MOON NIGHT](https://osu.ppy.sh/community/forums/topics/1775136). If you are somehow not familiar with the inspiration behind this skin, [*A FOOL MOON NIGHT*](https://osu.ppy.sh/beatmapsets/524026#osu/1141084) is an extremely popular 2016 beatmap set that inspired Spoo so much that they decided to draw **all** the skin elements present in the skin in the style of it's background! Such dedication has been noticed by the community and **1096** of them decided to award their votes to them, pushing them to the 3rd place.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="" frameborder="0" allowfullscreen></iframe>
+</div>
+
+![](https://assets.ppy.sh/profile-badges/SOTY2023-Fool-Moon-Night@2x.jpg)
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/Windosu.jpg)
+
+An unexpected issue has occurred, causing this top 10 list to crash. Error code: N05T@LG1@_0V3RL0@D. [Miorine](https://osu.ppy.sh/users/12630408) harnessed the full power of the holy trinity — early 2000s nostalgia, an extremely popular Windows operating system, and creativity. With those three combined, [Windosu XP](https://osu.ppy.sh/community/forums/topics/1790353) was crafted, and it caught the attention of the community right off the bat. Regardless of the fact that this is only the 2nd skin ever published by Miorine, they perfectly captured the nostalgia of many players which netted them a whopping **1511** votes, almost 400 more than 3rd place!
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="" frameborder="0" allowfullscreen></iframe>
+</div>
+
+![](https://assets.ppy.sh/profile-badges/SOTY2023-Windosu-XP@2x.jpg)
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/Whitecat.jpg)
+
+Everyone thought they were real funny commenting "rafis 2018-03-26 HDDT", or something along those lines. Well as it turns out, a skin with a similar amount of value, both historical and meme-wise, won this competition... Yup, [cyperdark](https://osu.ppy.sh/users/9893708)'s [WhiteCat 3.0](https://osu.ppy.sh/community/forums/topics/1864774) has been elected as **the** Skin of the Year, with **1574** votes! There isn't too much to say about the skin itself — it's centred around the top player [WhiteCat](https://osu.ppy.sh/users/4504101), it's clean, and it comes in a billion different variations to suit the needs of as many players as possible.
```

Comment:
can you explain what rafis has to do with this

Reply:
Check the first post and the top comment

Reply:
just to clarify since i cant access the osu website on my work's network:
![image](https://github.com/ppy/osu-wiki/assets/24534022/1ba73c4d-aca5-48c2-922a-21fb0f8e551e)